src/cookbook/lists/long-lists.md

```diff
@@ -98,12 +104,14 @@ class MyApp extends StatelessWidget {
 }
 ```
 
+## Children's extent
+
+To specify each item's extent, we can use either `itemExtent` or `prototypeItem`. Specifying either of them is more efficient than letting the children determine their own extent because the scrolling machinery can make use of the foreknowledge of the children's extent to save work, for example when the scroll position changes drastically.
+
 <noscript>
   <img src="/assets/images/docs/cookbook/long-lists.gif" alt="Long Lists Demo" class="site-mobile-screenshot" />
 </noscript>
 
-
-
-[`List.generate`]: {{site.api}}/flutter/dart-core/List/List.generate.html
-[`ListView`]: {{site.api}}/flutter/widgets/ListView-class.html
-[`ListView.builder`]: {{site.api}}/flutter/widgets/ListView/ListView.builder.html
+[`list.generate`]: {{site.api}}/flutter/dart-core/List/List.generate.html
+[`listview`]: {{site.api}}/flutter/widgets/ListView-class.html
+[`listview.builder`]: {{site.api}}/flutter/widgets/ListView/ListView.builder.html
```

Comment:
I don't understand these changes. The original is correct. My suggestion is to remove this change.

```suggestion
[`List.generate`]: {{site.api}}/flutter/dart-core/List/List.generate.html
[`ListView`]: {{site.api}}/flutter/widgets/ListView-class.html
[`ListView.builder`]: {{site.api}}/flutter/widgets/ListView/ListView.builder.html
```